kfdef/kfctl_aws.yaml

```diff
@@ -7,417 +7,98 @@ spec:
   - kustomizeConfig:
       repoRef:
         name: manifests
-        path: application/application-crds
-    name: application-crds
-  - kustomizeConfig:
-      overlays:
-      - application
-      repoRef:
-        name: manifests
-        path: application/application
+        path: application/v3
     name: application
   - kustomizeConfig:
-      parameters:
-      - name: namespace
-        value: istio-system
-      repoRef:
-        name: manifests
-        path: istio-1-3-1/istio-crds-1-3-1
-    name: istio-crds
-  - kustomizeConfig:
-      parameters:
-      - name: namespace
-        value: istio-system
       repoRef:
         name: manifests
-        path: istio-1-3-1/istio-install-1-3-1
-    name: istio-install
+        path: stacks/aws/application/istio-1-3-1-stack
```

Comment:
For long term, it's better for us to use v3 application in root folder and construct our sub-stack directly without extra application concepts. It looks fine now.

Reply:
I think so, in the future, we need to get rid of these applications.